mas/mints/03.json

```diff
@@ -0,0 +1 @@
+{"name":"skill/zarathustra/plantation_station_abci:0.1.0","description":"The scaffold skill is a scaffold for your own skill implementation.","code_uri":"ipfs://bafybeif3vkbiq6n4om6h7pobolzomdip3w7djt2o2ysnbhg43x6nwrh6ta","image":"ipfs://bafybeifl6hizzlvld6uxpjaw4ej6wr6sw5muzmopa23se3gpcwqgfcjffy","attributes":[{"trait_type":"version","value":"0.1.0"}]}
```

Comment:
The description should match the content. Suggest you update in package and fix the hashes which are affected (including deps). Core_uri not on registry